doc: LP token guide (#93)
valory-xyz/autonolas-tokenomics: README.md

## Deployed Protocol
The list of addresses can be found [here](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/mainnet_addresses.json).

## LP Token Guide

> In order to enable OLAS-based LP tokens, it is advised to check with the following [list of instructions](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/LP_token_guide.md). 

Reviewer: ```suggestion
It is advised to check the following [list of instructions](https://github.com/valory-xyz/autonolas-tokenomics/blob/main/docs/LP_token_guide.md) before enabling OLAS-based LP tokens. 
```
